src/client/components/pages/Schedule/SchedulePage.tsx

```diff
@@ -216,9 +231,12 @@ const SchedulePage: FunctionComponent = () => {
 
   return (
     <>
-      <VerticalSpace>
+      <div aria-label="course filter buttons">
+        <span id="Course Filter Button">show/hide catalog prefixes</span>
```

Comment:
Per the [HTML spec][mdnhtml], the id field shouldn't contain white space. I'd suggest kebab case (`id="course-filter-buttons"`) instead. 

[mdnhtml]: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id